### ISSUE TITLE
_mouseDestroy not implemented

### ISSUE THREAD

**Issue by chumanfu** [furf/jquery-ui-touch-punch#124]
Hi

I have a problem with jquery ui selectable. If I make UL which is scrollable selectable and then called selectable('destroy') the item no longer scrolls.

If I add this to Touch Punch it works:

  var mouseProto = $.ui.mouse.prototype,
      _mouseInit = mouseProto._mouseInit,
      _mouseDestroy = mouseProto._mouseDestroy,
      touchHandled;

  mouseProto._mouseDestroy = function () {

```
var self = this;

// Un-Delegate the touch handlers to the widget's element
self.element
  .unbind('touchstart', $.proxy(self, '_touchStart'))
  .unbind('touchmove', $.proxy(self, '_touchMove'))
  .unbind('touchend', $.proxy(self, '_touchEnd'));

// Call the original $.ui.mouse destroy method
_mouseDestroy.call(self);
```

  };

Is there any reason why _mouseDestroy has never been implemented?
Is my code above okay to use?

Cheers

Chris

**Comment by furf** [comment]
The code looks good. I am doing some housecleaning this week, so I'll see about possible implementation.

**Comment by furf** [comment]
Fixed in 0.2.3. Thanks!
